validate/tests.py

```diff
@@ -0,0 +1,56 @@
+import json
+import lib50.crypto
+import os
+
+
+def check_payload(payload, expected_payload):
+    """Checks whether payload is what we expected."""
+
+    status_code = 200
+    for item in expected_payload:
+        # Check for equality
+        if not (item in payload and payload[item] == expected_payload[item]):
+            # If they're at least the same type, assume it's just naming issues
+            if type(payload[item]) != type(expected_payload[item]):
+                status_code = 400
+
+    return status_code
```

Comment:
The integrity of the data is assured, yes. Checking if the signature is valid just ensures the data was transferred from to entry script to the Flask app properly, though -- it doesn't actually compare the data to the expected values.

Reply:
Didn't change this in the latest commit @kzidane for the reasoning above. Similarly, kept the functions in a separate file for now.

Reply:
Good point. Would it be simpler if we just did string comparison on compacted JSONs? Just trying to keep things simple.

Reply:
Probably not, since (I believe) the payload includes a unique tag hash that's generated at the time of the shell script's execution. So this function to check only the static parts of the payload is likely necessary.